change/@ni-nimble-components-5528f3eb-a2ce-4e80-9df4-6ddb04f8b015.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I think the follow up work you mentioned in the PR description (moving away from the now-deprecated APIs) is indeed worth tracking in tech debt.

Reply:
Created https://github.com/ni/nimble/issues/1551